server.js

```diff
@@ -49,14 +49,17 @@ const validateHeaders = (headers) => {
     const lang_header = headers['accept-language-stt'].toLowerCase();
 
     // if the passed language contains anything different from two (eg. pt)
-    // or five (eg. pt-br) chars, we deny
-    if (lang_header.length !== 2 && lang_header.length !== 5) {
+    // or five (eg. pt-br) chars, or eleven (eg. cmn-Hans-CN)
+    // or six (eg.fil-PH)  we deny
+    if (lang_header.length !== 2 && lang_header.length !== 5 &&
+        lang_header.length !== 11 && lang_header.length !== 6) {
       return 'accept-language-stt';
```

Comment:
I didn't review this before, and I'm confused how this works. You're returning the literal string `'accept-language-stt'` if conditions are met, otherwise the `validateHeaders` function implicitly returns undefined? Is there a reason not to have this function return a boolean condition?

Reply:
The validator returns which condition wasn't met so we could inform that to the user/api.